Titan/Titan.csproj

```diff
@@ -36,6 +36,9 @@
   <PropertyGroup Condition="'$(OS)' == 'Unix'">
     <DefineConstants>__UNIX__</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup>
+    <ApplicationIcon>Logo.ico</ApplicationIcon>
```

Comment:
I'd love to have it reference the `Logo.ico` in the `Resources` directory instead of having two files in different places that are the same.